charts/karpenter/templates/configmap.yaml

```diff
@@ -10,8 +10,17 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 data:
+  "clusterName": "{{ .Values.clusterName }}"
```

Comment:
If we nested these helm params in a single bucket, we could just wire it through transparently without needing to change this each time.